virtualizarr/tests/test_integration.py

```diff
@@ -223,22 +239,27 @@ def test_kerchunk_roundtrip_concat(
             ds2 = ds.isel(time=slice(1460, None))
 
             # save it to disk as netCDF (in temporary directory)
-            air1_nc_path = tmp_path / "air1.nc"
-            air2_nc_path = tmp_path / "air2.nc"
+            air1_nc_path = str(tmp_path / "air1.nc")
+            air2_nc_path = str(tmp_path / "air2.nc")
             ds1.to_netcdf(air1_nc_path)
             ds2.to_netcdf(air2_nc_path)
 
             # use open_dataset_via_kerchunk to read it as references
+            parser = HDFParser()
+            store1 = obstore_local(air1_nc_path)
+            store2 = obstore_local(air2_nc_path)
```

Comment:
Is it not possible to use one store that points to the directory containing both files?

Reply:
It would be if you use a local store without a prefix and provide the full file path for operations